source/docs/software/vision-processing/identifying-and-processing-the-targets.rst

```diff
@@ -10,21 +10,21 @@ Additional Options
 
 This document walks through the approach used by the example code provided in LabVIEW (for PC or roboRIO), C++ and Java. In addition to these options teams should be aware of the following alternatives that allow for vision processing on the Driver Station PC or an on-board PC:
 
-1.  `RoboRealm <http://www.roborealm.com/>`_
-2.  SmartDashboard Camera Extension (programmed in Java, works with any robot language)
-3.  `GRIP <https://wpiroboticsprojects.github.io/GRIP/>`_
+1. `RoboRealm <http://www.roborealm.com/>`__
```

Comment:
No need for two underscores. 
```suggestion
1. `RoboRealm <http://www.roborealm.com/>`_
```

Reply:
The style guide says to use two underscores for links.  Should there be different guidance in there (eg 1 underscore some places and two in others)?

Reply:
That was due to two ares of conflicting documentation in regards to initially learning RST. Should be updated to only contain 1.

Reply:
For example, the Sphinx source contains two underscores, but their supporting documentation contains one.

Reply:
On second thought, according to this stack exchange post there's a reason for the difference. https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use